Leetcode/EigenDemo/main.cpp

```diff
@@ -7,19 +7,11 @@
 
 #include <iostream>
 
-#include "Eigen/core"
-#include "Eigen/Dense"
+#include "eigenPractice.hpp"
```

Comment:
```suggestion
#include "eigen_practice.hpp"
```

https://google.github.io/styleguide/cppguide.html#File_Names

Filenames should be all lowercase and can include underscores (\_) or dashes (-). Follow the convention that your project uses. If there is no consistent local pattern to follow, prefer "\_".